tests/k8s_env/ksp/ksp_test.go

```diff
@@ -528,7 +511,7 @@ var _ = Describe("Ksp", func() {
 			Expect(err).To(BeNil())
 
 			// /bin/bash -c pwd shoud passed as only /bin/ls is blocked by policy
-			sout, _, err = K8sExecInPod(ub1, "multiubuntu",
+			sout, _, err := K8sExecInPod(ub1, "multiubuntu",
 				[]string{"bash", "-c", "/bin/dash -c pwd"})
 			Expect(err).To(BeNil())
 			fmt.Printf("OUTPUT: %s\n", sout)
```

Comment:
We can move this to AssertCommand with eventually false, maybe keep the matcher to contain valid linux path or be * something like that